### PR TITLE
Recv: handle clientIQ

### PR DESCRIPTION
**Pull request by psilva261**
Hey!

As promised, here is another pull request regarding the C2S Ping! :D `Recv()` now can also handle the Pong response that is wrapped in a `<iq/>` tag, so I also created a new `type IQ`.

Greetings, Philip

### CODE REVIEW

File: xmpp.go [hunk 1]
```diff
@@ -585,6 +593,14 @@ func (c *Client) Recv() (stanza interface{}, err error) {
 			return Chat{Type: "roster", Roster: r}, nil
 		case *clientPresence:
 			return Presence{v.From, v.To, v.Type, v.Show, v.Status}, nil
+		case *clientIQ:
+			result := IQ{
```
Comment (ThomasBS) [inline xmpp.go]: Why not just follow the same code style and return the `result` directly? :)

**Comment by psilva261** [comment]
You're right, I'll update the pull request. :)

**Comment by mattn** [comment]
thank you

**Comment by psilva261** [comment]
:tada:

**Pull request merged**